client.go

```diff
@@ -2094,6 +2107,10 @@ func (c *Client) executeRequestMiddlewares(req *Request) (err error) {
 // Executes method executes the given `Request` object and returns
 // response or error.
 func (c *Client) execute(req *Request) (*Response, error) {
```

Comment:
```suggestion
 // execute method executes the given `Request` object and returns
 // response or error.
 func (c *Client) execute(req *Request) (*Response, error) {
```

Or you rename the method

Reply:
This should be done in a separate housekeeping PR, IMO this shouldn't be a part of my changes

Reply:
Indeed